version: '3'

services:
  jupyter:
    image: dataops:latest
    networks: ["hadoop"]
    env_file:
      - ./hadoop.env
    volumes:
      - notebooks:/notebooks
      - /c:/c
    ports:
     - 8888:8888
     - 4040:4040

networks:
  hadoop:
    external:
      name: 3_hadoop_default

volumes:
  notebooks:

